AppAuth-JS/built/app/index.d.ts

```diff
@@ -0,0 +1,46 @@
+import { AuthorizationServiceConfiguration } from '../authorization_service_configuration';
+/**
+ * The wrapper appication.
+ */
+export declare class App {
+    private authorizeUrl;
+    private tokenUrl;
+    private revokeUrl;
+    private logoutUrl;
+    private userInfoUrl;
+    private clientId;
+    private clientSecret;
+    private redirectUri;
+    private scope;
+    private postLogoutRedirectUri;
+    private discoveryUri;
+    private userStore;
+    private flowTypeInternal;
+    private notifier;
+    private authorizationHandler;
+    private configuration;
+    constructor({ authorizeUrl, tokenUrl, revokeUrl, logoutUrl, userInfoUrl, flowType, userStore, clientId, clientSecret, redirectUri, scope, postLogoutRedirectUri, discoveryUri }?: {
```

Comment:
format this line

Reply:
generated file. all built/ folder files are generated